cpp/include/cudf/detail/stream_compaction.hpp

```diff
@@ -95,17 +96,19 @@ std::unique_ptr<table> distinct(
  * generated. If there are duplicate rows, which index is kept depends on the `keep` parameter.
  *
  * @param input The input table
- * @param keep Get index of the first, last, any, or none row among the found duplicates rows
+ * @param keep Get index of any, or the first, last, or none row among the found duplicates rows
```

Comment:
Align this with the docs in `stream_compaction.hpp`.
```suggestion
 * @param keep Get index of any, first, last, or none of the found duplicates
```